pkg/base1/test-http.js

```diff
@@ -25,12 +25,12 @@ const test_server = {
     port: parseInt(window.location.port, 10)
 };
 
-QUnit.test("simple request", function (assert) {
+QUnit.test("simple request", assert => {
     const done = assert.async();
-    assert.expect(2);
+    assert.expect(1);
```

Comment:
hmm, what does 1 and 2 mean?

Reply:
It's the number of assertions that a test expects to run. As I dropped the `assert.equal(this.state(), ...`, I had to reduce it by one.